doc/languages-frameworks/python.section.md

```diff
@@ -1261,15 +1306,16 @@ are used in `buildPythonPackage`.
 - `pythonCatchConflictsHook` to check whether a Python package is not already existing.
 - `pythonImportsCheckHook` to check whether importing the listed modules works.
 - `pythonRemoveBinBytecode` to remove bytecode from the `/bin` folder.
+- `pythonRelaxDepsHook` will relax Python dependencies restrictions for the package.
+  See [example usage](#using-pythonrelaxdepshook).
 - `setuptoolsBuildHook` to build a wheel using `setuptools`.
 - `setuptoolsCheckHook` to run tests with `python setup.py test`.
+- `sphinxHook` to build coumentation and manages using Sphinx.
```

Comment:
documentation, s/manages/manage? (I'm no native speaker, but I think it is infinitive form here)

Reply:
I meant to write manpages there.

Reply:
Heh, sorry, was in "non-conscious" review mode :C

Reply:
```suggestion
- `sphinxHook` to build documentation and man pages using Sphinx.
```